OmiseSDK/Client.swift

```diff
@@ -138,6 +138,113 @@ import os
         })
         dataTask.resume()
     }
+
+    public func retrieveChargeStatusWithCompletionHandler(from tokenID: String, completionHandler: (((ChargeStatus, Error?)) -> Void)?) {
```

Comment:
Is it possible to reuse `completeRequest()` function?
https://github.com/omise/omise-ios/blob/7058a0e24ec86f47ea9eac142719c6603eb62f90/OmiseSDK/Client.swift#L310

Reply:
It can't for now. Because `ChargeStatus` is not conform `Object` protocol. 🙇

Reply:
I just wonder about it because the function looks very similar. Anyway, please skip my comment.